internal/cli/kraft/cloud/volume/remove/remove.go

```diff
@@ -36,16 +39,25 @@ func Remove(ctx context.Context, opts *RemoveOptions, args ...string) error {
 
 func NewCmd() *cobra.Command {
 	cmd, err := cmdfactory.New(&RemoveOptions{}, cobra.Command{
-		Short:   "Permanently delete a persistent volume",
-		Use:     "remove UUID [UUID [...]]",
-		Args:    cobra.MinimumNArgs(1),
-		Aliases: []string{"rm"},
+		Short:   "Remove volumes",
+		Use:     "remove [FLAGS] [UUID|NAME [UUID|NAME]...]",
+		Args:    cobra.MinimumNArgs(0),
+		Aliases: []string{"rm", "delete"},
 		Long: heredoc.Doc(`
-			Permanently delete a persistent volume.
+			Permanently delete persistent volumes.
```

Comment:
Just kill the `Long` attribute; as it contains no more information than the `Short`.